src/Akeneo/Pim/Structure/Bundle/Resources/workspaces/settings-ui/src/pages/categories/CategoryEditPage.tsx

```diff
@@ -100,7 +100,9 @@ const CategoryEditPage: FC = () => {
     const rootCategory = category && category.root ? category.root : category;
 
     setCategoryLabel(
-      category && category.labels.hasOwnProperty(uiLocale) ? category.labels[uiLocale] : `[${category.code}]`
+      category && category.labels.hasOwnProperty(uiLocale) && category.labels[uiLocale]
+        ? category.labels[uiLocale]
+        : `[${category.code}]`
```

Comment:
You can also use getLabel function from `@akeneo-pim-community/shared` :

```suggestion
      getLabel(category.labels ?? {}, uiLocale, category.code)
```

Category cannot be not here (early return)

Reply:
Indeed it's better like this. Thanks !